test/src/test/java/ai/lzy/test/scenarios/PyApiTest.java

```diff
@@ -22,6 +22,11 @@ public class PyApiTest {
         CondaEnvironment.reconfigureConda(false);  // To optimize conda configuration
     }
 
+    @Test
+    public void testOleg() {
```

Comment:
it's fanny, but absolutely unclear what this is about :)

Reply:
it should be removed:)